### PR TITLE
Add miRNA dataset loader

### PR DESCRIPTION
**Pull request by qanastek**
All tests passed correctly.

### CODE REVIEW

**Comment by qanastek** [comment]
Issue https://github.com/bigscience-workshop/biomedical/issues/53

File: biodatasets/miRNA/miRNA.py [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+MicroRNAs (miRNAs) have demonstrated their potential as post-transcriptional gene expression
+regulators, participating in a wide spectrum of regulatory events such as apoptosis,
+differentiation, and stress response. Apart from the role of miRNAs in normal physiology,
+their dysregulation is implicated in a vast array of diseases. Dissection of miRNA-related
+associations are valuable for contemplating their mechanism in diseases, leading to the
+discovery of novel miRNAs for disease prognosis, diagnosis, and therapy.
+"""
+
+_HOMEPAGE = "https://www.scai.fraunhofer.de/en/business-research-areas/bioinformatics/downloads/download-mirna-test-corpus.html"
+_LICENSE = "Attribution-NonCommercial 3.0 Unported (CC BY-NC 3.0)"
```
Comment (galtay) [inline biodatasets/miRNA/miRNA.py:89]: can you share where this license decision came from please?

File: biodatasets/miRNA/miRNA.py [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+"""
+
+_DESCRIPTION = """\
+MicroRNAs (miRNAs) have demonstrated their potential as post-transcriptional gene expression
+regulators, participating in a wide spectrum of regulatory events such as apoptosis,
+differentiation, and stress response. Apart from the role of miRNAs in normal physiology,
+their dysregulation is implicated in a vast array of diseases. Dissection of miRNA-related
+associations are valuable for contemplating their mechanism in diseases, leading to the
+discovery of novel miRNAs for disease prognosis, diagnosis, and therapy.
+"""
```
Comment (galtay) [inline biodatasets/miRNA/miRNA.py:86]: this sounds like a description of miRNAs, but not necessarily a description of the dataset. maybe check here for a snippet or quote https://www.scai.fraunhofer.de/content/dam/scai/de/downloads/bioinformatik/miRNA/README.txt

Comment (galtay) [inline biodatasets/miRNA/miRNA.py:86]: this sounds like a description of miRNAs, but not necessarily a description of the dataset. maybe check here for a snippet or quote https://www.scai.fraunhofer.de/content/dam/scai/de/downloads/bioinformatik/miRNA/README.txt

File: biodatasets/miRNA/miRNA.py [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+    http://www.scai.fraunhofer.de/mirna-corpora.html.
+},
+note={26535109[pmid]},
+note={PMC4602280[pmcid]},
+issn={2046-1402},
+url={https://pubmed.ncbi.nlm.nih.gov/26535109},
+language={eng}
+}
+"""
+
```
Comment (galtay) [inline biodatasets/miRNA/miRNA.py:78]: looks like the `_DATASETNAME` variable is missing. is there an example script in which this is missing? If so I can fix that. also can we please rename using lowercase to follow the pattern set by the other datasets (e.g. `mirna`)

Comment (galtay) [inline biodatasets/miRNA/miRNA.py:78]: looks like the `_DATASETNAME` variable is missing. is there an example script in which this is missing? If so I can fix that. also can we please rename using lowercase to follow the pattern set by the other datasets (e.g. `mirna`)

**Review by galtay — CHANGES_REQUESTED** [review]

File: biodatasets/miRNA/miRNA.py (hunk 3 above)
Comment (qanastek) [reply biodatasets/miRNA/miRNA.py:78]: Only two example have this missing variable :

- https://github.com/bigscience-workshop/biomedical/blob/master/examples/bc5cdr.py
- https://github.com/bigscience-workshop/biomedical/blob/master/examples/bioasq.py

File: biodatasets/miRNA/miRNA.py (hunk 2 above)
Comment (qanastek) [reply biodatasets/miRNA/miRNA.py:86]: What do you think of that ?

````plain
The corpus consists of 301 Medline citations. The documents were screened for mentions of miRNA in the abstract text.
Gene, disease and miRNA entities were manually annotated. The corpus comprises of two separate files, a train and a test set, coming from 201 and 100 documents respectively. 
````

This is closer to `BC5CDR` description.

File: biodatasets/miRNA/miRNA.py (hunk 1 above)
Comment (qanastek) [reply biodatasets/miRNA/miRNA.py:89]: I found the license terms in the following [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4602280/#__sec15) linked to the corpus.

The specific section is :

```plain
Data associated with the article are available under the terms of the Creative Commons Attribution-NonCommercial Licence, which permits non-commercial use, distribution, and reproduction in any medium, provided the original data is properly cited. http://creativecommons.org/licenses/by-nc/3.0/

Corpora availability: http://www.scai.fraunhofer.de/mirnacorpora.html
```

**Comment by qanastek** [comment]
I fill up the dataset card in the Issue https://github.com/bigscience-workshop/biomedical/issues/53#issuecomment-1086604262.

**Comment by qanastek** [comment]
I have commit the changes under the Commit 036bec25b40e9d7cfe582f6a5165de919814d16e.

**Review by galtay — CHANGES_REQUESTED** [review]
Looks good. Can you also make the dataset directory and loader script name lowercase please.

**Comment by qanastek** [comment]
Done! Sorry, I update it locally, but my git default configuration had the settings `core.ignorecase` set to true.

**Review by galtay — APPROVED** [review]

**Comment by qanastek** [comment]
Does the `Merging is blocked - The base branch requires all commits to be signed. ` status need any interaction from me ?

**Comment by galtay** [comment]
> Does the `Merging is blocked - The base branch requires all commits to be signed. ` status need any interaction from me ?

Still debating internally. No action needed from you at the moment. Will let you know soon (and will probably make an announcement in the discord)

**Comment by galtay** [comment]
🎉

**Pull request merged**